packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/VariableStarViewer/VariableStarViewerContainer.spec.js

```diff
@@ -557,4 +559,34 @@ describe('Component > VariableStarViewerContainer', function () {
       }).then(done, done)
     })
   })
+
+  describe('with pan and zoom per scatter plot module', function () {
+    let wrapper
+    before(function () {
+      const subject = Factory.build('subject', {
+        locations: [
+          { 'application/json': 'http://localhost:8080/variableStar.json' },
+          { 'image/png': 'http://localhost:8080/image1.png' }
+        ]
+      })
+      wrapper = shallow(
+        <VariableStarViewerContainer
+          subject={subject}
+        />
+      )
+    })
+
+    it('should default to not have pan and zoom enabled for either scatter plot', function () {
+      expect(wrapper.state().allowPanZoom).to.equal('')
```

Comment:
It would be safer to test the prop that's passed down from state here too.

Reply:
The spec on line 590 is testing the props being passed down.

Reply:
I know. Props should be tested here so that the test doesn't depend on whether the tested component is a class or a function.
```suggestion
      expect(wrapper.find(VariableStarViewer).props().allowPanZoom).to.equal('')
```